server.js

```diff
@@ -17,6 +17,6 @@ app.use(
 );
 
 // Serve the files on port 3000.
-app.listen(3000, () => {
-  console.log('Example app listening on port 3000!\n');
-});
\ No newline at end of file
+// app.listen(3000, () => {
+//   console.log('Example app listening on port 3000!\n');
+// });
```

Comment:
- [ ] I think it would be better to remove unused code from your project rather than comment it out to keep your code clean and follow [the best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)💯

![image](https://user-images.githubusercontent.com/45231276/181750281-0d99a210-25c7-4b54-b455-141811a9c361.png)